### ISSUE TITLE
Add light mode and dark mode 

### ISSUE THREAD

**Issue by arpittiwari24** [arpittiwari24/YourNextSaas.online#1]
Currently the ui is all dark mode . There should be a button somewhere in the navbar to switch themes .

**Comment by 1-SubhamSingh** [comment]
Hey there, 
I am willing to complete this task , can you assign this to me.
It would be better from your side if you add the gssoc-extnd 2024 tag    :)

**Comment by arpittiwari24** [comment]
Yup sure , I am assigning this to you  . Also , have added the tag  :)

**Comment by toufiqfarhan0** [comment]
i would like to work on this
